pkg/term/proxy_test.go

```diff
@@ -112,4 +112,53 @@ func TestEscapeProxyRead(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Equal(t, nr, len(keys), fmt.Sprintf("nr should be equal to %d", len(keys)))
 	assert.DeepEqual(t, keys, buf)
+
+	escapeKeys, _ = ToBytes("ctrl-c,ctrl-z")
```

Comment:
This test becomes really unwieldy with all the test-cases (some of which look to depend on values set by previous test-cases).

Looks like they all _slightly_ differ (so we cannot easily change it into test-table, but we can use subtests to group them into logical test-cases, which would make this test easier to read

Reply:
Went ahead and did some refactoring; https://github.com/moby/moby/pull/39800 (you can update the new tests to use the same / similar pattern)

Reply:
Sounds good, should I cherry-pick your commit in my PR, or do you expect it to be merged first and then I'll rebase?

Reply:
Hope it will be merged soon, but you could already modify the new tests to match

Reply:
Great, I've pulled your changes and will rebase once they're merged.